lib/network/api/api.go

```diff
@@ -24,23 +23,27 @@ const (
 	GetTransactionByHashHandlerPattern     = "/transactions/{id}"
 	GetTransactionOperationsHandlerPattern = "/transactions/{id}/operations"
 	PostTransactionPattern                 = "/transactions"
+	GetNodeInfoPattern                     = "/"
 )
 
 type NetworkHandlerAPI struct {
-	localNode *node.LocalNode
-	network   network.Network
-	storage   *storage.LevelDBBackend
-	urlPrefix string
-	version   string
+	localNode      *node.LocalNode
+	network        network.Network
+	storage        *storage.LevelDBBackend
+	urlPrefix      string
+	version        string
+	nodeInfo       node.NodeInfo
+	GetLatestBlock func() block.Block
```

Comment:
:+1:

Reply:
Why do you need this ? It looks to be only used for tests

Reply:
Check this, https://github.com/bosnet/sebak/pull/534/files#diff-c908837bca0e8ac0569de07b175175f6R24 .